docs/source/deprecations.rst

```diff
@@ -2,6 +2,13 @@
 Deprecation timeline
 ====================
 
+From 5.x.x to 6.0.0
```

Comment:
Why not 5.0.0? Next version will be a major with this change, so we already know we will go from 5.0.0 to 6.0.0

Reply:
mmm yes, you're right